browser/resources/settings/brave_overrides/privacy_page.ts

```diff
@@ -42,20 +43,27 @@ function InsertGoogleSignInSubpage (
       console.error(
         '[Brave Settings Overrides] Couldn\'t find Google signin subpage')
     } else {
-      googleSignInSubpage.setAttribute(
-          'page-title',
+      const pageTitle =
+        googleSignInTemplate.content.getElementById('page-title')
+      if (!pageTitle) {
+        console.error(
+          '[Brave Settings Overrides] Couldn\'t find Google sigin page-title')
+      } else {
+        let pageTitleNode = document.createTextNode(
           loadTimeData.getString('siteSettingsCategoryGoogleSignIn'))
+        pageTitle.appendChild(pageTitleNode)
+      }
       const googleSignInDefault =
         googleSignInTemplate.content.getElementById('googleSignInDefault')
       if (!googleSignInDefault) {
         console.error(
           '[Brave Settings Overrides] Couldn\'t find Google signin default')
       } else {
         googleSignInDefault.setAttribute(
-            'toggle-off-label',
+            'block-option-label',
             loadTimeData.getString('siteSettingsGoogleSignInAsk'))
```

Comment:
This should now be `siteSettingsGoogleSignInBlock`, the two labels were previously the same because it was a toggle and it was confusing whether the toggle represented current state or what would happen if you changed the toggle.

Reply:
Great catch, fixed.